sdk/healthinsights/azure-healthinsights-radiologyinsights/README.md

```diff
@@ -76,31 +76,154 @@ Sample code snippets are provided to illustrate using long-running operations [b
 
 ## Key concepts
 
+Once you've initialized a 'RadiologyInsightsClient', you can use it to analyse document text by displaying inferences found within the text.
+* Age Mismatch
+* Laterality Discrepancy
+* Sex Mismatch
+* Complete Order Discrepancy
+* Limited Order Discrepancy
+* Finding
+* Critical Result
+* Follow-up Recommendation
+* Communication
+* Radiology Procedure
 Radiology Insights currently supports one document from one patient. Please take a look [here][inferences] for more detailed information about the inferences this service produces. 
```

Comment:
> - Radiology Procedure Radiology Insights currently supports one document from one patient. Please take a look [here](https://learn.microsoft.com/azure/azure-health-insights/radiology-insights/inferences) for more detailed information about the inferences this service produces.

line 90 gets concatenated to previous list item.

Reply:
Fixed